source/fundamentals/connection/network-compression.txt

```diff
@@ -53,7 +51,7 @@ by specifying the algorithms in one of two ways:
    .. tab:: MongoClientOptions
       :tabid: mongoclientoptions
 
-      To enable compression using the `MongoClientOptions <{+api+}/api/global.html#MongoClientOptions>`__,
+      To enable compression using `MongoClientOptions <{+api+}/api/global.html#MongoClientOptions>`__,
```

Comment:
I: [Avoid "using" on its own](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u)

S: 

```suggestion
      To enable compression with `MongoClientOptions <{+api+}/api/global.html#MongoClientOptions>`__,
```

Reply:
Thanks for identifying this. I think "by using" might be better. I avoid "with" because it usually signals the explanation isn't as specific as it could/should be. E.g.
"Connect to MongoDB with the driver" vs "Connect to MongoDB by using the appropriate method in the driver".